apisix/core/utils.lua

```diff
@@ -260,6 +260,18 @@ function _M.gethostname()
 end
 
 
+function _M.get_file(file_name)
```

Comment:
Could we put it into core/io.lua and let core.request refer it?

Reply:
updated